container-storage-setup.sh

```diff
@@ -34,8 +34,8 @@ STORAGE_IN_FILE="/etc/sysconfig/docker-storage-setup"
 STORAGE_OUT_FILE="/etc/sysconfig/docker-storage"
 STORAGE_DRIVERS="devicemapper overlay overlay2"
 
-PIPE1=/run/css-fifo1
-PIPE2=/run/css-fifo2
+PIPE1=/run/css-$$-fifo1
+PIPE2=/run/css-$$-fifo2
```

Comment:
Sorry, I just noticed that we already have logic to cleanup pipes.

trap cleanup_pipes EXIT
cleanup_pipes(){
    rm -f $PIPE1
    rm -f $PIPE2
    rmdir $TEMPDIR 2>/dev/null
}

So you can get rid of newly added code and push patch again.